src/dmd/typesem.d

```diff
@@ -1494,7 +1494,11 @@ extern(C++) Type typeSemantic(Type type, const ref Loc loc, Scope* sc)
             // Now that we completed semantic for the argument types,
             // run semantic on their default values,
             // bearing in mind tuples have been expanded.
-            size_t tupleStartIdx = size_t.max;
+            // We need to keep a pair of [oidx, eidx] (original index,
+            // extended index), so we now to run semantic when the original
```

Comment:
grammar. get rid of `to`.

Reply:
Good catch. Actually the sentence wasn't quite accurate so I rephrased it.